docs/platform/reference/end-of-life.md

```diff
@@ -56,3 +56,34 @@ April 30, 2025. For further assistance, contact
 To keep your InfluxDB data, download a backup before April 30, 2025. You can use the
 Aiven Console, Aiven API, or Aiven CLI to download the data. For detailed instructions,
 see [Download InfluxDB data](/docs/products/metrics/howto/download-influxdb-data).
+
+## Aiven for Caching
+
+**EOL date**: April 30, 2025
+
+### Service impact
+
+- **End of availability (EOA)**: December 31, 2024
+  After this date, you can no longer create new Aiven for Caching services. Existing
+  services continue to operate until the end of life (EOL) date.
+
+- **End of life (EOL)**: April 30, 2025
+  On this date, all active Aiven for Caching services are automatically upgraded
+  to **Aiven for Valkey** to maintain Redis compatibility. Any powered-off Aiven for
+  Caching services are permanently deleted, making data from these services
+  inaccessible.
+
+### Migration options
+
+The recommended alternative to Aiven for Caching is **Aiven for Valkey**. Aiven for
+Valkey is fully compatible with Aiven for Caching, allowing your existing applications
+and workflows to operate without modification. Essential settings, including DNS, URLs,
+ACLs, and user configurations, remain unchanged, ensuring minimal impact on your
+infrastructure and workflows.
+
+For upgrade instructions, see
+[Upgrade from Aiven for Caching to Aiven for Valkey](/docs/products/caching/howto/upgrade-aiven-for-caching-to-valkey).
+
+To ensure uninterrupted service, complete your migration to Aiven for Valkey before
+April 30, 2025. For further assistance, contact the
+[Aiven support team](mailto:support@aiven.io) or your [sales team](mailto:sales@aiven.io).
```

Comment:
Do you still want to use this sales email? Consider replacing it as follows:
```suggestion
[Aiven support team](mailto:support@aiven.io) or your account team.
```

Reply:
I have always used 'sales team' with the email, as that was also mentioned in the ticket. However, I have updated it to 'account team' here as suggested.